### PR TITLE
arcticdb 3.0.0

### PR DESCRIPTION
**Pull request by poodlewars**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "3.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 149660905f2a1169a8bfeeb0f58e35e496747589a7d2147252181c4e826b3683
+  sha256: 22745af39d7be461dc5ba13e6afd25e1bad5709673d78a1f609708f70c3cd711
```
Comment (jjerphan) [inline recipe/meta.yaml]:
```suggestion
  sha256: 6d2d68e8b931d8a871f9034eaabe48acdfe3578d7c5862122dce119b7d867947
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 149660905f2a1169a8bfeeb0f58e35e496747589a7d2147252181c4e826b3683
+  sha256: 6d2d68e8b931d8a871f9034eaabe48acdfe3578d7c5862122dce119b7d867947
 
   patches:
```
Comment (jjerphan) [inline recipe/meta.yaml:12]: Side-note: the `patches` section can be removed since https://github.com/man-group/ArcticDB/issues/662 has been merged.

**Review by jjerphan — COMMENTED** [review]
@conda-forge-admin, please rerender

**Review by jjerphan — APPROVED** [review]
LGTM given 🟢 CI checks.

Thank you, @poodlewars.

I will rename the title of this PR for consistency with the previous PRs'.

**Review by jjerphan — CHANGES_REQUESTED** [review]
Since https://github.com/man-group/ArcticDB/pull/761 which introduce the dependence of `pymongo` is part of this release, `pymongo` must also be added to the `run` requirements.

**Comment by phoebusm** [comment]
@jjerphan `pymongo` dependency is for testing only. It should not be required in the `run` requirements.

**Review by jjerphan — DISMISSED** [review]
OK then LGTM.

We just need to document this line [update this line](https://github.com/man-group/ArcticDB/blob/a491bb03416e5dd419691abe0da096bb1127e89e/environment_unix.yml#L68).

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: Some libraries of of the Azure SDK from C++ need to be added to `host`, see https://github.com/conda-forge/arcticdb-feedstock/pull/56/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR69-R71.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -54,6 +54,8 @@ requirements:
     - cyrus-sasl =2.1.27
     - msgpack-c
     - aws-sdk-cpp
+    - azure-identity-cpp
```
Comment (jjerphan) [inline recipe/meta.yaml:57]: We also need to explicitly add `azure-core-cpp` for now.
```suggestion
    - azure-core-cpp
    - azure-identity-cpp
```

**Review by jjerphan — APPROVED** [review]
LGTM given :green_circle: CI checks.

**Pull request merged**